architecture/design/docdb-change-data-capture.md

```diff
@@ -203,7 +203,7 @@ WAIT: This option causes the command to wait if the slot is active until it beco
 
 - Removes a Replication Slot. A publication can only be dropped by its superuser or a user with replication privileges.
 - The WAIT option is treated differently In YugabyteDB. An “active” replication slot means a slot which has been consumed for a certain timeframe.
-- We will define this timeframe via a GFlag ysql_replication_slot_activity_threshold with a default of 5 minutes.
+- We will define this timeframe via a GFlag ysql_cdc_active_replication_slot_window_ms with a default of 5 minutes.
```

Comment:
I think it should be possible to add a link to the GFlags under `yb-tserver.md` file for the added GFlag.

Reply:
Thanks for suggesting. I've added the description of the GFlag.

Unfortunately, we can't link it here because this is in the design section. At the moment, I can link it to the `yb_tserver.md` of the `preview` version of the docs but ideally I would have liked to not hardcode it to the `preview`.

I don't see any other GFlag being linked in the `design` section. Let's keep it as it is